shogun-lib/src/main/java/de/terrestris/shogun/lib/controller/ApplicationController.java

```diff
@@ -32,4 +42,82 @@
     description = "The endpoints to manage applications"
 )
 @SecurityRequirement(name = "bearer-key")
-public class ApplicationController extends BaseController<ApplicationService, Application> { }
+@Log4j2
+public class ApplicationController extends BaseController<ApplicationService, Application> {
+    @GetMapping("/name/{name}")
+    @ResponseStatus(HttpStatus.OK)
+    @Operation(security = { @SecurityRequirement(name = "bearer-key") })
+    @ApiResponses(value = {
+        @ApiResponse(
+            responseCode = "200",
+            description = "Ok: The entity was successfully created"
```

Comment:
Should be something like:

```suggestion
            description = "Ok: Successfully returned the application with the given name"
```